app/components/Form/ProjectEmissionIntensityReportForm.tsx

```diff
@@ -82,61 +81,61 @@ export const createEmissionIntensityReportUiSchema = (
-  if (viewOnly)
-    uiSchemaCopy.teimpReporting.productionFunctionalUnit["ui:label"] = "";
+  if (viewOnly) uiSchemaCopy.productionFunctionalUnit["ui:label"] = "";
 
-  uiSchemaCopy.teimpReporting.baselineEmissionIntensity["ui:options"] = {
-    ...uiSchemaCopy.teimpReporting.baselineEmissionIntensity["ui:options"],
+  uiSchemaCopy.baselineEmissionIntensity["ui:options"] = {
+    ...uiSchemaCopy.baselineEmissionIntensity["ui:options"],
     contentSuffix: contentSuffix(
       emissionFunctionalUnit,
       productionFunctionalUnit
     ),
   };
-  uiSchemaCopy.teimpReporting.targetEmissionIntensity["ui:options"] = {
-    ...uiSchemaCopy.teimpReporting.targetEmissionIntensity["ui:options"],
+  uiSchemaCopy.targetEmissionIntensity["ui:options"] = {
+    ...uiSchemaCopy.targetEmissionIntensity["ui:options"],
     contentSuffix: contentSuffix(
       emissionFunctionalUnit,
       productionFunctionalUnit
     ),
   };
-  uiSchemaCopy.teimpReporting.postProjectEmissionIntensity["ui:options"] = {
-    ...uiSchemaCopy.teimpReporting.postProjectEmissionIntensity["ui:options"],
+  uiSchemaCopy.postProjectEmissionIntensity["ui:options"] = {
+    ...uiSchemaCopy.postProjectEmissionIntensity["ui:options"],
     contentSuffix: contentSuffix(
       emissionFunctionalUnit,
       productionFunctionalUnit
     ),
   };
-  uiSchemaCopy.teimpReporting.totalLifetimeEmissionReduction["ui:options"] = {
-    ...uiSchemaCopy.teimpReporting.totalLifetimeEmissionReduction["ui:options"],
+  uiSchemaCopy.totalLifetimeEmissionReduction["ui:options"] = {
+    ...uiSchemaCopy.totalLifetimeEmissionReduction["ui:options"],
     contentSuffix: contentSuffix(emissionFunctionalUnit),
   };
 
-  uiSchemaCopy.teimpReporting.measurementPeriodEndDate["ui:options"] = {
-    ...uiSchemaCopy.teimpReporting.measurementPeriodEndDate["ui:options"],
+  uiSchemaCopy.measurementPeriodEndDate["ui:options"] = {
+    ...uiSchemaCopy.measurementPeriodEndDate["ui:options"],
     contentSuffix: reportDurationSuffix,
   };
 
-  uiSchemaCopy.teimpReporting.emissionFunctionalUnit["ui:options"] = {
-    ...uiSchemaCopy.teimpReporting.emissionFunctionalUnit["ui:options"],
+  uiSchemaCopy.emissionFunctionalUnit["ui:options"] = {
+    ...uiSchemaCopy.emissionFunctionalUnit["ui:options"],
     contentSuffix: emissionFunctionalUnitSuffix(),
   };
 
-  /**
-   * We need to set the ui:FieldTemplate to the EmptyFieldTemplate
-   * and ui:ObjectFieldTemplate to the EmptyObjectFieldTemplate
-   * to avoid showing field labels and outlines on parent object.
-   * There is room for improvement on how this is done
-   * but for now it is the best solution.
-   */
-  uiSchemaCopy["ui:FieldTemplate"] = ReadOnlyEmptyFieldTempalte;
-  uiSchemaCopy["ui:ObjectFieldTemplate"] = EmptyObjectFieldTemplate;
-
   return uiSchemaCopy;
 };
 
 const ProjectEmissionsIntensityReport: React.FC<Props> = (props) => {
   const formRefs: MutableRefObject<{}> = useRef({});
 
+  const { eiFormBySlug } = useFragment(
```

Comment:
If I get the schema via the formChanges instead, sometimes I get the emission_intensity schema, but sometimes I get the reporting_requirement schema (this is what caused some of the fields to disappear; when I got the reporting_requirement schema instead). Not sure why this was happening, but querying for the schema we want should fix it